source/exercises/mailroom-part1.rst

```diff
@@ -0,0 +1,106 @@
+.. _exercise_mailroom_part1:
+
+
+Part 1: Establishing The Data Structure (Lesson 3)
```

Comment:
This title is what shows up in the index, so maybe just 

Mailroom Part 1.

And I don't think we need the (Lesson 3) anymore -- that was there so it wasn't too confusing when it was all in one file.

Reply:
will check on this.